### ISSUE TITLE
Termux's proot-distro package has updated to Ubuntu hirsute (instead of Ubuntu 20.01)

### ISSUE THREAD

**Issue by WheezyE** [lowspecman420/AnBox86#7]
Hey man :)

Just wanted to let you know that proot-distro seems to have removed Ubuntu 20.01 and moved to a newer distro just as of last week.  This would be an easy fix except that [proot-static](https://github.com/ZhymabekRoman/proot-static) seems to conflict with the new update somehow.

The main issue is that I can log into proot with `proot-distro login --isolated ubuntu` just fine and run things if I don't have proot-static downloaded, but if I do download proot-static then I can't do `apt update` anymore inside ubuntu.

I'm trying to see if I can find a workaround but I'm stuck at the moment

I suspect that the main issue might be the apt repo's not having armhf packages or something

**Comment by WheezyE** [comment]
Very small update:
 - Installing proot from proot-distro then logging into proot and running arch shows `aarch64`
 - Installing proot-static in Termux then logging into proot and running arch shows `armv8l`

**Comment by ZhymabekRoman** [comment]
Closed in: https://github.com/ZhymabekRoman/proot-static/commit/f65dc6d469cb08028a1ce303eb5e749fe48756e8

**Comment by WheezyE** [comment]
Thank you again to ZhymabekRoman!

Just updating this thread to say I've gotten farther now with the new proot-distro (now incorporated into new proot-static), but there's a new proot-distro error now with creating user accounts. I've opened a ticket [here](https://github.com/termux/proot-distro/issues/159) to try to resolve it with proot-distro.

**Comment by WheezyE** [comment]
> there's a new proot-distro error now with creating user accounts

Xeffyr from proot-distro suggested this might be a bug in proot, so I opened another issue there: https://github.com/termux/proot/issues/191

**Comment by WheezyE** [comment]
I forgot to mention that the user account problem has been solved in proot now, though I’m getting a few more snags now (https://github.com/termux/proot/issues/193).  I’ll update this thread when I know more or hear back from the proot guys

**Comment by WheezyE** [comment]
This sudo error was fixed https://github.com/termux/proot/issues/193 although I'm still getting PTRACE errors with box86 now.  I'll keep testing

**Comment by WheezyE** [comment]
Sorry this took so long to fix. We got some regressions in Termux fixed though. And in the process we figured out how to run box86 on a multiarch PRoot that also runs box64.  This means that we don't need a 32-bit proot anymore to run box86, which is pretty cool - though it also means that we won't be needing proot-static anymore.  I think we wouldn't have gotten to this stage at all if not for ZhymabekRoman's contributions and efforts with the 32-bit proot-static though.

I'm going to close this issue now and also make a pull request for an AnBox86_64.sh that can install box86/box64 & wine/wine64 side-by-side.
